docs/getting-started/cli/subsembly-spec.md

```diff
@@ -16,27 +16,140 @@ subsembly spec [to] [src] [raw] [wasm]
 
 #### Parameters
 
-* **`--to`** - \[optional\] initialisation path for the new chain specification file
-* **`--src`** - \[optional\] source specification file to convert to raw, defaults to current directory
-* **`--raw`** - \[optional\] path to save converted raw chain specification file. Defaults to **`{srcPath}/raw-chain-spec.json`**
-* **`--wasm`** - \[optional\] path to the compiled hex encoded **`wasm`** binary. Defaults to **`build/subsembly-wasm`**
+* **`--to`** - optional parameter, determines initialisation path for the new chain specification file, defaults to current directory
+* **`--src`** - optional parameter, determines source specification file to convert to raw, defaults to current directory
+* **`--raw`** - optional parameter, determines path to save converted raw chain specification file. Defaults to **`{path-of-src}/raw-chain-spec.json`**
+* **`--wasm`** - optional parameter, determines path to the compiled hex encoded **`wasm`** binary. Defaults to **`build/subsembly-wasm`**
 
 ### Examples
 
-```text
+{% tabs %}
+{% tab title="1st" %}
+```bash
 # Generating new chain spec file
-subsembly spec
+subsembly-user new-sub % subsembly spec
+Successfully generated new custom spec file!
+```
+{% endtab %}
 
+{% tab title="2nd" %}
+```bash
 # or (it is recommended to put all your spec files inside spec-files/ folder in root directory)
```

Comment:
This information seems strange to me. It is confusing. Should we include such information in the "terminal"? I think that it might be better if we put it outside the `terminal` view